.vsts-dotnet-ci.yml

```diff
@@ -1,29 +1,36 @@
-queue:
+pool:
   name: Hosted VS2017
   demands: msbuild
 
+strategy:
+  matrix:
+    Debug:
+      BuildConfiguration: Debug
+    Release:
+      BuildConfiguration: Release
+
 variables:
   "Parameters.solution": MSBuildLocator.sln
-  BuildConfiguration: Release
   SignType: Test
-  BuildPlatform: ''
 
 steps:
-- task: MSBuild@1
-  displayName: Set version number
-  condition: eq(variables['system.pullrequest.isfork'], false)
+- task: DotNetCoreInstaller@0
```

Comment:
I can't seem to find notes on `DotNetCoreInstaller` in the [tasks documentation](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/index?view=azure-devops)

Reply:
Should this possibly be UseDotNet?

https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/dotnet-core-tool-installer?view=azure-devops

Reply:
Yup, looks like a recent rename according to https://github.com/microsoft/azure-pipelines-tasks/pull/10044#issuecomment-481055325.

This does work: https://github.com/microsoft/azure-pipelines-tasks/tree/0a63e0e6c725bc7aeb6609e07123843267ef3a7e/Tasks/DotNetCoreInstallerV0

but yeah, I'll move over to the new one.